apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-docs.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PlatformActionBarDocsComponent } from './platform-action-bar-docs.component';
+
+describe('PlatformActionBarDocsComponent', () => {
+    let component: PlatformActionBarDocsComponent;
+    let fixture: ComponentFixture<PlatformActionBarDocsComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [PlatformActionBarDocsComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PlatformActionBarDocsComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
```

Comment:
Missing tests

Reply:
Removed, as documentation not require unit testing